packages/contracts/contracts/GenArt721CoreV3.sol

```diff
@@ -160,6 +161,8 @@ contract GenArt721CoreV3 is
     /// Art Blocks Project ID range: [3-373]
     address public constant ART_BLOCKS_ERC721TOKEN_ADDRESS_V1 =
         0xa7d8d9ef8D8Ce8992Df33D8b8CF4Aebabd5bD270;
+    /// pointer to next contract associated with this contract
```

Comment:
Do you think it's worth re-iterating that this is the next `core` contract? 

```
address public nextCoreContract;
```

Reply:
good call! updated in [011ea8c](https://github.com/ArtBlocks/artblocks-contracts/pull/1480/commits/011ea8c84ea909a40648f80a933ac726d8ef8216)